Misc/NEWS.d/next/Library/2024-07-02-05-08-16.gh-issue-61103.TMxkuM.rst

```diff
@@ -0,0 +1,4 @@
+Support :c:expr:`float complex` and :c:expr:`long double complex` C types in
+:mod:`ctypes` via :class:`~ctypes.c_float_complex` and
+:class:`~ctypes.c_longdouble_complex` if compiler has C11 complex
+arithmetic. Patch by Sergey B Kirpichev.
```

Comment:
You may edit the entry of your previous PR (complete the previous NEWS file), instead of adding a second Changelog entry;